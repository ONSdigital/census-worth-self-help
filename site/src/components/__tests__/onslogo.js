import React from "react"
import renderer from "react-test-renderer"

import OnsLogo from "../onslogo"

describe("ONS Logo", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<OnsLogo/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})