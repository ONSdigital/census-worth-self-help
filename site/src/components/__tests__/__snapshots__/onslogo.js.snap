// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ONS Logo renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "k6unfh",
      "next": undefined,
      "styles": "
        width: 100%;
        max-width: 800px;
      ",
    }
  }
>
  <div
    style={
      Object {
        "border": 0,
        "clip": "rect(0 0 0 0)",
        "height": "1px",
        "margin": "-1px",
        "overflow": "hidden",
        "padding": 0,
        "position": "absolute",
        "width": "1px",
      }
    }
  >
    <h1>
      Self Help Facility
    </h1>
  </div>
  <img
    alt="Main logo"
    css={
      Object {
        "map": undefined,
        "name": "10mfl7n",
        "next": undefined,
        "styles": "
          padding: 0px 15px 20px 15px;
        ",
      }
    }
    data-test="ons-logo__header"
    src="/ONS_logo.svg"
  />
</div>
`;
